hawc/apps/mgmt/views.py

```diff
@@ -46,12 +47,31 @@ class RobTaskMixin:
     def get_rob_queryset(self, RiskOfBias):
         raise NotImplementedError("Abstract method; requires implementation")
 
+    def set_study_ids(self, rob_qs: None):
```

Comment:
I wanted to make it so that `get_review_tasks` and `get_review_studies` can be called independently of each other. However, this mixin probably isn't going to be used in any other way than it currently is, so we can probably rely on `get_review_tasks` being run before `get_review_studies`, so I agree with your comment. It's been simplified in commit #[91c15c0](https://github.com/shapiromatron/hawc/pull/629/commits/91c15c0ce759bf77e1acd0140d0974d612740fac).